website/static/js/fangorn.js

```diff
@@ -1009,6 +1020,34 @@ function _removeEvent (event, items, col) {
     }
 }
 
+function doCheckout(item, checker, error_message) {
```

Comment:
"checker" implies that a function is supposed to be passed. How about "checkout"?